pages/index.tsx

```diff
@@ -334,27 +345,26 @@ export default function Home({ featuredStorefronts }: HomeProps) {
       <CenteredContentCol>
         <Section>
           <Marketing xs={22} md={16}>
-            <HeroTitle>Discover, explore, and collect NFTs from incredible creators on Solana</HeroTitle>
-            <Pitch>
-              Tools built by creators, for creators, owned by creators.
-            </Pitch>
+            <HeroTitle>
+              Discover, explore, and collect NFTs from incredible creators on Solana
+            </HeroTitle>
+            <Pitch>Tools built by creators, for creators, owned by creators.</Pitch>
             <Space direction="horizontal" size="large">
-              <Button onClick={() => connect()}>
-                Create Your Store
-              </Button>
+              <Button onClick={() => connect()}>Create Your Store</Button>
             </Space>
           </Marketing>
           <Col xs={24} md={8}>
             <Text strong>Featured Listings</Text>
-            {loading ? (
-              <SkeletonListing />
-            ) : (
-              <HeroCarousel autoplay={true} dots={{ className: 'carousel-dots' }} dotPosition="top">
-                {listings.slice(0, 5).map((listing) => (
-                  <ListingPreview key={listing.listingAddress} {...listing} />
-                ))}
-              </HeroCarousel>
-            )}
+            <HeroCarousel autoplay={true} dots={{ className: 'carousel-dots' }} dotPosition="top">
+              {featuredListings.map((listing) => (
+                <ListingPreview key={listing.listingAddress} {...listing} />
```

Comment:
What I as with the listing preview is that the price shows as 0 while loading. I did like the styling of skeleton listing.

Also personally find the existing approach easier to understand. No need to play 5 empty shells. Team project though so if others like this style I'm good with it.